domains/azhammakestrash.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "AzhamProdLive",
+        "email": "websitedomain.regroup588@passinbox.com"
+        // This is a alias address which is redirected to my actual address
```

Comment:
this line is making the .json file invalid

Reply:
I tried to make it as a comment but I guess I'm doing it wrong, I just fixed it